Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorTransformComponentSelection.cpp

```diff
@@ -3037,6 +3038,9 @@ namespace AzToolsFramework
             // Update when the transform mode changes.
             m_actionManagerInterface->AddActionToUpdater(TransformModeChangedUpdaterIdentifier, actionIdentifier);
 
+            // This action is only accessible outside of Component Modes
+            m_actionManagerInterface->AssignModeToAction(DefaultActionContextModeIdentifier, actionIdentifier);
```

Comment:
suggestion: I wonder if wrapping this in a function with a friendlier name might make this easier to use and understand, e.g....

```
// This action is only accessible outside of Component Modes
void MakeActionUnavilableInComponentMode(
    ActionManagerInterface* actionManagerInterface, const ActionIdentifier actionIdentifier)
{
    actionManagerInterface->AssignModeToAction(DefaultActionContextModeIdentifier, actionIdentifier);
}
```

Reply:
I will mark it down and do a pass (since many actions had this added in previous PRs)